oak_attestation/src/dice.rs

```diff
@@ -180,113 +170,99 @@ impl TryFrom<DiceData> for DiceBuilder {
             .eca_private_key;
         let signing_key = SigningKey::from_slice(eca_private_key).map_err(anyhow::Error::msg)?;
 
+        // Zero out the ECA private key if it was set.
+        if let Some(certificate_authority) = &mut value.certificate_authority {
+            certificate_authority.eca_private_key.zeroize();
+        }
+
         Ok(DiceBuilder {
             evidence: evidence.clone(),
             signing_key: signing_key.clone(),
```

Comment:
Is this signing key also zeroized? It does not seem so, but I think it should?

Reply:
Will create a separate PR, also to rename the signing key to `eca_private_key` because right now it may be confusing (since we have a signing key for signing arbitrary data)